includes/wizards/class-dashboard.php

```diff
@@ -57,10 +57,10 @@ protected function get_dashboard() {
 			[
 				'slug'        => 'site-design',
 				'name'        => esc_html__( 'Site Design', 'newspack' ),
-				'url'         => '#',
+				'url'         => admin_url( 'customize.php' ),
 				'description' => esc_html__( 'Branding, color, typography, layouts', 'newspack' ),
 				'image'       => Newspack::plugin_url() . '/assets/wizards/dashboard/site-design-icon.svg',
-				'status'      => 'disabled',
+				'status'      => true,
```

Comment:
I think we'll probably get rid of this `status` option in the future since all the cards will always be enabled, but right now `true` is a weird option for it (even though it works). `'enabled'` and `'disabled'` are currently the two officially recognized statuses.

Reply:
Good call. Done in https://github.com/Automattic/newspack-plugin/pull/250/commits/d56905c55e140b6d0efa3715640e0bb69404018d